gips/data/landsat/landsat.py

```diff
@@ -620,6 +627,9 @@ class landsatData(Data):
     }
     __toastring = 'toa: use top of the atmosphere reflectance'
     __visible_bands_union = [color for color in Asset._sensors['LC8']['colors'] if 'LWIR' not in color]
+
+    # note C1 products are also C1S3 products; they're added in below
+    # TODO don't use manual tables of repeated information in the first place
```

Comment:
DRY is good, but we also need a human readable version  of this information.  Maybe `gips_info ${DRIVER} --products` could dump the products dictionary or something like that.